### PR TITLE
lp-160 : add layout

### PR DESCRIPTION
**Pull request by lduranteau**
### JIRA link
https://companieshouse.atlassian.net/browse/LP-160

### Change description
Add layout templates

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes look good on mobile
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: src/views/includes/piwik-scripts.njk [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<script src="{{MATOMO_ASSET_PATH}}/javascripts/app/piwik-enable.js"></script>
+<script type="application/javascript">
+    const journeyTypes = {
+        "register-an-overseas-entity": "Register",
```
Comment (markpit) [inline src/views/includes/piwik-scripts.njk:4]: still has overseas-entitiy references

File: src/views/layout.njk [hunk 2]
```diff
@@ -0,0 +1,98 @@
+{% extends "govuk/template.njk" %}
+
+{% from "govuk/components/back-link/macro.njk"           import govukBackLink %}
```
Comment (markpit) [inline src/views/layout.njk:3]: i would remove the govUk components, we won't know what we need yet

Comment (lduranteau) [reply src/views/layout.njk:3]: you mean all imports?

Comment (markpit) [reply src/views/layout.njk:3]: line 3 - 22, we can add them in when we need them otherwise we might be importing things we don't need.

**Pull request merged**